packages/web-components/fast-foundation/src/select/select.template.ts

```diff
@@ -23,16 +23,16 @@ export const SelectTemplate: ViewTemplate<Select> = html`
         <div
             aria-activedescendant="${x => (x.open ? x.ariaActiveDescendant : null)}"
             aria-controls="listbox"
-            aria-expanded="${x => x.ariaExpanded}"
             aria-haspopup="listbox"
+            aria-labelledby="selected-value"
```

Comment:
This DOM element was requiring a label (or `aria-labelledby`) in order to be a11y compliant

Reply:
I think this is fine, but we should run this via ARIA 1.2 rather than 1.1 as the implications are significant. Is there a way for you to validate against ARIA 1.2 in AXE instead of 1.1?